local-cluster/tests/local_cluster.rs

```diff
@@ -1984,6 +1984,122 @@ fn test_future_tower_master_slave() {
     do_test_future_tower(ClusterMode::MasterSlave);
 }
 
+#[test]
+#[serial]
+fn test_hard_fork() {
```

Comment:
there was no test or whatever for hard fork code.. hence this generic test name can be justified.